shared/js/background/utils.es6.js

```diff
@@ -182,8 +182,12 @@ function isBrokenList (url, lists) {
 }
 
 // We inject this into content scripts
-function getBrokenCanvasScriptList () {
-    return tdsStorage?.fingerprinting?.canvas?.scripts || []
+function getBrokenScriptLists () {
+    const brokenScripts = {};
+    for (let key in tdsStorage?.fingerprinting) {
+        brokenScripts[key] = tdsStorage.fingerprinting[key]?.scripts || [];
+    }
```

Comment:
Yeah, do you mind if we readdress this in my current task instead though? The cost is going to be less than a ms on current size and I'd rather not risk breaking code that I think is pretty robust now.

`initStringExemptionLists` is also more expensive than it should be too given that we pass args into all three functions now rather than having an `init` function that then calls them all a bit smarter.